lib/amqp_client.js

```diff
@@ -192,18 +192,22 @@ AMQPClient.prototype.createSender = function(address, options) {
     throw new Error('Must connect before creating links');
   }
 
-  address = address || this._defaultQueue;
+  address = u.parseLinkAddress(address || this._defaultQueue);
```

Comment:
We'd talked about not calling parseLinkAddress if defaultSubjects is false, because it will do the wrong thing.